chromium_src/components/error_page/common/localized_error.cc

```diff
@@ -0,0 +1,33 @@
+#include "components/url_formatter/url_formatter.h"
+#include "net/base/escape.h"
+#include "url/gurl.h"
+
+namespace brave_error_page {
+
+// Hardcode these constants to avoid dependencies on //chrome and //content.
+// Note: This is following what has been done in original source of this file.
+// unfortunately, the original kChromeUIScheme variable is in an anonymous
+// namespace.
+const char kChromeUIScheme[] = "chrome";
```

Comment:
pls use `error_page` namespace. you should be able to access `kChromeUIScheme` from anonymous NS in this file, this can be done by forward-declaring `GetFailedUrlString`, including upstream file and then implementing the function.

Reply:
Cool, thanks! That's much nicer. As I'm sure you can tell my C++ is pretty rusty. With regard to style, is it better to do the forward declaration from a header file, or to just do the forward declaration before the macro?